src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs
+    .find(tab => tab.id === selectedTabId)
+    || tabs[0];
+
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${currentTab.title}`}
+      </h1>
+
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                className={cn({ 'is-active': tab.id === selectedTabId })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (tab.id !== selectedTabId) {
+                      onTabSelected(tab);
+                    }
```

Comment:
Move this logic to the helper function and use it here

Reply:
Done!